src/Avalonia.Controls/Flyouts/PopupFlyoutBase.cs

```diff
@@ -205,7 +205,7 @@ protected virtual bool HideCore(bool canCancel = true)
 
             OnClosed();
 
-            Target = null;
+            Popup.PlacementTarget = Target = null;
```

Comment:
Popup PlacementTarget is already cleaned in latest master https://github.com/AvaloniaUI/Avalonia/pull/15358/files#diff-9326460a9b8237ad899529c7ac3a503a84bc6e43d9ba697eb3fae9f1698015a7L192-R193

Reply:
Ah okay :)